src/queue/Queue.cxx

```diff
@@ -270,6 +270,7 @@ Queue::Clear() noexcept
 	}
 
 	length = 0;
+	last_loaded_playlist = "";
```

Comment:
This has a smell - `last_loaded_playlist.clear();` is better and faster

Reply:
Replaced